profiles/models.py

```diff
@@ -288,3 +293,17 @@ class Education(models.Model):
     school_city = models.TextField()
     school_state_or_territory = models.TextField()
     school_country = models.TextField()
+
+    def __str__(self):
+        degree_title = ''
```

Comment:
Suggestion: `degree_title = dict(self.DEGREE_CHOICES).get(self.degree_name, '')` would simplify this.